### ISSUE TITLE
You have missing dependencies! spyder_kernels >=2.0.1;<2.1.0 : 2.1.0 (NOK)

### ISSUE THREAD

**Issue by akmalkady** [spyder-ide/spyder#15387]
## Problem Description
When I start spyder 5.0 or 5.1 I got this error:
```
You have missing dependencies!


# Mandatory:
spyder_kernels >=2.0.1;<2.1.0 : 2.1.0 (NOK)

Please install them to avoid this message.

Note: Spyder could work without some of these dependencies, however to have a smooth experience when using Spyder we strongly recommend you to install all the listed missing dependencies.

Failing to install these dependencies might result in bugs. Please be sure that any found bugs are not the direct result of missing dependencies, prior to reporting a new issue.

```

## Versions

* Spyder version: 
5.0 and 5.1
* Python version: 
3.8
* Qt version: 
* PyQt version: 
* Operating System name/version: 
Linux mint 19

**Comment by akmalkady** [comment]
I solved this issue by doing the following:
```
cd ./ak_env/lib/python3.8/site-packages
nano spyder_kernels/_version.py
```

Then I changed 
`VERSION_INFO = (2, 1, 0)`
to:
`VERSION_INFO = (2, 0, 1)`

And the message gone!

I wrote it here so if it is a bug you can fix it.

**Comment by ccordoba12** [comment]
Hey @akmalkady, thanks for reporting. This is a duplicate of issue #15069 and it's already fixed in our 5.0.1 version, which should be soon part of Anaconda.
